pkg/controlplane/apiserver/options/options.go

```diff
@@ -239,7 +239,9 @@ func (o *Options) Complete(ctx context.Context, fss cliflag.NamedFlagSets, alter
 	}
 
 	// put authorization options in final state
-	completed.Authorization.Complete()
+	if completed.Authorization != nil {
+		completed.Authorization.Complete()
+	}
```

Comment:
Potentially quick upstream change and we can drop this.

Reply:
We can actually drop this already, [`(*BuiltInAuthorizationOptions).Complete()`](https://github.com/kubernetes/kubernetes/blob/v1.32.0/pkg/kubeapiserver/options/authorization.go#L87-L91) already checks if the caller is `nil`.

In v1.31.0, this was indeed not being checked: https://github.com/kubernetes/kubernetes/blob/v1.31.0/pkg/kubeapiserver/options/authorization.go#L87-L91